mesonbuild/dependencies/configtool.py

```diff
@@ -63,14 +72,12 @@ def _sanitize_version(self, version):
             return m.group(0).rstrip('.')
         return version
 
-    def find_config(self, versions: T.Optional[T.List[str]] = None, returncode: int = 0) \
-            -> T.Tuple[T.Optional[str], T.Optional[str]]:
+    def find_config(self, versions: T.List[str] = None, returncode: int = 0) \
+            -> T.Tuple[T.Optional[T.List[str]], T.Optional[str]]:
```

Comment:
Did you look through the ConfigToolDependency subclasses to make sure that they're all going to be okay with this change?

Reply:
Yes. `find_config` is only called once in `ConfigToolDependency` and only overwritten once in `GnuStepDependency`.

However, I did forget to remove the `= None` for the `versions` kwarg...